Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDllHelpers/DebuggeFlowController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using DbgEngManaged;
+using System;
+
+namespace CsDebugScript.Engine.Debuggers.DbgEngDllHelpers
+{
+    /// <summary>
+    /// Controler for Debugee actions during live debugging.
+    /// </summary>
+    class DebuggeeFlowController
```

Comment:
Internal is default right?

---
In reply to: [69980965](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69980965) [](ancestors = 69980965)